etc/kayobe/ansible/templates/wazuh-secrets.yml.j2

```
@@ -0,0 +1,18 @@
+---
+{%- set wazuh_admin_pass = secrets_wazuh.wazuh_admin_pass | default(lookup('password', '/dev/null'), true) -%}
+{%- set wazuh_user_pass = secrets_wazuh.wazuh_user_pass | default(lookup('password', '/dev/null'), true) -%}
+
+
+# Secrets used by Wazuh managers and agents
+# Store these securely and use lookups here
+secrets_wazuh:
+  # Wazuh agent authd pass
+  authd_pass: "{{ secrets_wazuh.authd_pass | default(lookup('password', '/dev/null'), true) }}"
+  # Strengthen default wazuh api user pass
+  wazuh_api_users:
+    - username: "wazuh"
+      password: "{{ secrets_wazuh.wazuh_api_users[0].password | default(lookup('password', '/dev/null length=30' ), true) }}"
+  # Elasticsearch 'admin' user pass
+  opendistro_admin_password: "{{ secrets_wazuh.opendistro_admin_password | default(lookup('password', '/dev/null'), true) }}"
+  # Elasticsearch 'kibanaserver' user pass
+  opendistro_kibana_password: "{{ secrets_wazuh.opendistro_kibana_password | default(lookup('password', '/dev/null'), true) }}"
```

Should the comments say `OpenSearch` rather than `Elasticsearch`
```suggestion
  # OpenSearch 'admin' user pass
  opendistro_admin_password: "{{ secrets_wazuh.opendistro_admin_password | default(lookup('password', '/dev/null'), true) }}"
  # OpenSearch 'kibanaserver' user pass
  opendistro_kibana_password: "{{ secrets_wazuh.opendistro_kibana_password | default(lookup('password', '/dev/null'), true) }}"
```

---

Fixed. Thanks for commit suggestion!